src/main/java/co/rsk/federate/config/PowpegNodeConfigParameter.java

```diff
@@ -0,0 +1,51 @@
+package co.rsk.federate.config;
+
+import java.util.Objects;
+import java.util.function.Function;
+
+public enum PowpegNodeConfigParameter {
+  FEDERATOR_ENABLED("federator.enabled", Boolean.toString(false)),
+  PEGOUT_ENABLED("federator.pegout.enabled", Boolean.toString(false)),
```

Comment:
Were these 2 false by default? I feel like they should be true

Reply:
Yes `federator.pegout.enabled` was true before but not `federator.enabled`. I have updated both of them to be true.